content/templates/subscription-form.pug

```diff
@@ -6,52 +6,51 @@ block body
             .u-spacer-bottom
                 img(src="images/logo.svg" alt="Smart")
             .c-hr
-            h1.c-h2 Inscription
+            h1.c-h2 Registration
             .c-content
-                p Pour une utilisation optimale des services que Smart met à vote disposition, il est vivement souhaitable et recommandé de participer à une session Smart : <a href="#">mode d'emploi</a>.
-
+                p For optimal use of the Smart services, it is highly recommended to attend a Smart session: <a href="#">user manual</a>.
             .u-spacer-bottom
                 +c-alert({
                     title: "Important",
-                    message: `<ul><li>Vous recevrez VOS codes personnels par mail pour accéder à votre « compte Smart » (et encoder vos contrats) dans les 48h ouvrés suivant l'envoi de vos documents d'inscription. N'hésitez pas à contacter le bureau Smart le plus proche de chez vous en cas d'urgence.</li><li>N'oubliez pas de toujours déclarer vos dates de travail à l'avance. Il n'est pas permis d'antidater vos prestations.</li>`,
+                    message: `<ul><li>You will receive your personal codes to access your « “Smart account” » (and to encode your contracts)  by email within 48 working hours of sending your registration documents. Do not hesitate to contact the nearest Smart office in case of emergency.</li><li>Remember to always declare your working dates in advance. It is not allowed to backdate your services.</li>`,
```

Comment:
- There are two kinds of quotes here, I guess this is a mistake.
- Even in french, "encode" is a bit weird. I suggest "fill".
- There is a double space before "by".
- I'm wondering if "dates" could be changed for "days" (I guess it doens't matter too much).

Reply:
Thanks for the feedback!
I updated it in this commit 527752882f0c916dc686a3a10b6b68e5d71740c0